var/spack/repos/builtin/packages/py-cudf/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+
+
+class PyCudf(PythonPackage):
+    """Built based on the Apache Arrow columnar memory format,
+    cuDF is a GPU DataFrame library for loading, joining,
+    aggregating, filtering, and otherwise manipulating data."""
+
+    homepage = "https://rapids.ai"
+    url      = "https://github.com/rapidsai/cudf/archive/v0.15.0.tar.gz"
+
+    version('0.15.0',  sha256='57b8036134368daa8a967eae971ee121cdc58c8678a03eba1358ed2b927dc16e')
+
+    build_directory = 'python/cudf'
+
+    depends_on('cmake@3.14:', type='build')
+    depends_on('python@3.6:3.7', type=('build', 'run'))
```

Comment:
I would remove this line. The code likely works with Python 3.8 even if they haven't actually tested it.